### PR TITLE
Data/course registry

### PR DESCRIPTION
**Pull request by 19521178**

### CODE REVIEW

File: SqlServer/data.sql [hunk 1]
```diff
@@ -168,6 +168,10 @@ CREATE TABLE SubjectClass
   Period NVARCHAR(MAX) NOT NULL,
   WeekDay NVARCHAR(MAX) NOT NULL,
   IdThumbnail UNIQUEIDENTIFIER NOT NULL,
+  IdTraningForm UNIQUEIDENTIFIER NOT NULL,
+  Code NVARCHAR(MAX) NOT NULL,
+  NumberStudent INT NOT NULL,
```
Comment (vinhqngo5) [inline SqlServer/data.sql:173]: NumberOfStudents

File: SqlServer/data.sql [hunk 2]
```diff
@@ -168,6 +168,10 @@ CREATE TABLE SubjectClass
   Period NVARCHAR(MAX) NOT NULL,
   WeekDay NVARCHAR(MAX) NOT NULL,
   IdThumbnail UNIQUEIDENTIFIER NOT NULL,
+  IdTraningForm UNIQUEIDENTIFIER NOT NULL,
+  Code NVARCHAR(MAX) NOT NULL,
+  NumberStudent INT NOT NULL,
+  MaxOfRegister INT NOT NULL,
```
Comment (vinhqngo5) [inline SqlServer/data.sql:174]: MaxNumberOfStudents

File: StudentManagement/StudentManagement/Components/CourseRegistry/SubjectRegistryDataGrid.xaml.cs [hunk 3]
```diff
@@ -36,15 +37,15 @@ public SubjectRegistryDataGrid()
         }
 
 
-        public ObservableCollection<CourseRegistryItem> Data
+        public ObservableCollection<CourseItems> Data
         {
-            get { return (ObservableCollection<CourseRegistryItem>)GetValue(DataProperty); }
+            get { return (ObservableCollection<CourseItems>)GetValue(DataProperty); }
```
Comment (vinhqngo5) [inline StudentManagement/StudentManagement/Components/CourseRegistry/SubjectRegistryDataGrid.xaml.cs:42]: CourseItem

**Review by lehuutrung1412 — COMMENTED** [review]
Reviewed

**Pull request merged**